sycl/test-e2e/syclcompat/launch/launch.cpp

```diff
@@ -21,6 +21,9 @@
  **************************************************************************/
 // https://github.com/intel/llvm/issues/14387
 // UNSUPPORTED: gpu-intel-dg2
+// https://github.com/intel/llvm/issues/14826
+// XFAIL: arch-intel_gpu_pvc
```

Comment:
(@intel/syclcompat-lib-reviewers) I'd say we're probably OK to disable these tests for PVC for now, but is there a plan to re-enable them in time? I'm asking because we've had a couple of such cases recently & I'm a little twitchy about accruing too much tech debt. Cheers

Reply:
I don't think the XFAIL here creates new debt, just highlights the existing hidden one :)

Reply:
> (@intel/syclcompat-lib-reviewers) I'd say we're probably OK to disable these tests for PVC for now, but is there a plan to re-enable them in time?
I will create internal JIRAs for all the disable tests and will assign them to the respective teams. That way, we can keep track of the disabled tests.

Reply:
I think the XFAIL from this test can be removed. The code that run into issues was refactored in #14441 to `sycl/test-e2e/syclcompat/launch/launch_policy_lmem.cpp` and was removed from this file.